demo/gallery_component_events/run.py

```diff
@@ -0,0 +1,24 @@
+import gradio as gr 
+
+with gr.Blocks() as demo:
+    cheetahs = [
+        "https://upload.wikimedia.org/wikipedia/commons/0/09/TheCheethcat.jpg",
+        "https://nationalzoo.si.edu/sites/default/files/animals/cheetah-003.jpg",
+        "https://img.etimg.com/thumb/msid-50159822,width-650,imgsize-129520,,resizemode-4,quality-100/.jpg",
+        "https://nationalzoo.si.edu/sites/default/files/animals/cheetah-002.jpg",
+        "https://images.theconversation.com/files/375893/original/file-20201218-13-a8h8uq.jpg?ixlib=rb-1.1.0&rect=16%2C407%2C5515%2C2924&q=45&auto=format&w=496&fit=clip",
```

Comment:
I would suggest using images from our S3 bucket so that our functional test doesn't become flaky

Reply:
yea good point!

Reply:
Made the change and pushed up!